tests/test_config.py

```diff
@@ -14,8 +15,15 @@ def cleanup(cls):
         if os.path.exists(iniFile):
             os.remove(iniFile)
 
+    @classmethod
+    def update_setting_ini_as_firsttime(cls):
+        config = configs._prod_config.config
+        config.set('DEFAULT', 'firsttime', 'yes')
+        configs._prod_config.update_config()
+
     @classmethod
     def setUpClass(cls):
+        cls.update_setting_ini_as_firsttime()
```

Comment:
What's the purpose of init the config and remove it immediately?

Reply:
reset the content of config file because one case requires this

Reply:
Then why do you immediately remove it?

Reply:
Then why do you immediately remove it?

Reply:
updated